R/readIEA_acOwnership.R

```diff
@@ -0,0 +1,31 @@
+#' Read AC ownership rates per household from IEA Sources
+#'
+#' @author Hagen Tockhorn
+#'
+#' @importFrom dplyr select mutate rename
+#' @importFrom tidyr pivot_longer
+#' @importFrom quitte as.quitte
+#' @importFrom magclass as.magpie
+
+readIEA_acOwnership <- function() { #nolint object_name_linter
+
+  data <- read.csv2("iea_ac-ownership-rates.csv")
+
+  colnames(data) <- gsub("\\.", " ", colnames(data)) %>%
+    tolower()
```

Comment:
Please try this but I think you can avoid the dots in the first place.
```suggestion
  data <- read.csv2("iea_ac-ownership-rates.csv", check.names = FALSE)

  colnames(data) <- tolower(colnames(data))
```